public/js/popup.js

```diff
@@ -58,21 +83,30 @@ function warningModal(clone, options) {
         const errorCell = line.insertCell(2);
         errorCell.classList.add("highlight");
         errorCell.classList.add("msg");
+
         const positionCell = line.insertCell(3);
         positionCell.classList.add("position");
+        if (kind !== "short-identifiers" && kind !== "obfuscated-code") {
+            positionCell.addEventListener("mouseenter", (event) => fetchCodeLine(event.srcElement, unpkgFile, location, cache, lineId))
```

Comment:
```suggestion
            const currLocation = kind === "encoded-literal" ? location[0] : location;
            positionCell.addEventListener("mouseenter", (event) => fetchCodeLine(event.srcElement, unpkgFile, currLocation, cache, lineId));
```

Because encoded-literal warning has Array of SourceLocation for now we take only the first row (we may have to improve this in the futur).

Reply:
Oh, I forgot about this 😨 ! We discuss it on the discord channel, sorry!